labelbox/client.py

```diff
@@ -1385,6 +1396,27 @@ def get_catalog_slice(self, slice_id) -> CatalogSlice:
         res = self.execute(query_str, {'id': slice_id})
         return Entity.CatalogSlice(self, res['getSavedQuery'])
 
+    def unarchive_root_feature_schema_node(
```

Comment:
New unarchive client method reaches out to the api's rest endpoint to unarchive the feature schema in the ontology.